soccer/gameplay/robocup-py.cpp

```diff
@@ -255,15 +272,26 @@ void State_draw_arc(SystemState* self, const Geometry2d::Arc* arc,
     self->drawArc(*arc, Color_from_tuple(rgb), QString::fromStdString(layer));
 }
 
+// TODO(ashaw596) Fix this lie of a function
```

Comment:
drawLine vs drawSegment. Infinite lines vs segments. idk what to do about these semantics when they're everywhere. Slight semantic clarification and hard to track down python bugs or just leaving it slightly confusing with naming.